spec/controllers/api/v1/products_controller_spec.rb

```diff
@@ -0,0 +1,128 @@
+require 'rails_helper'
+
+describe Api::V1::ProductsController do
+  describe '[GET] #index' do
+    before do
+      create_list(:product, 5)
```

Comment:
IMO 5 objects is a little exaggeration 😄 Usually we do not want to create too much objects in test db. In case where we want to explicitly test collections 2 objects is enough 👌